tests/test_multiannotator.py

```diff
@@ -0,0 +1,187 @@
+    sizes=[80, 40, 40],
+    avg_trace=0.8,
+    num_annotators=50,
+    seed=1,  # set to None for non-reproducible randomness
+):
+    np.random.seed(seed=seed)
+
+    m = len(means)  # number of classes
+    n = sum(sizes)
+    local_data = []
+    labels = []
+    test_data = []
+    test_labels = []
+
+    for idx in range(m):
+        local_data.append(
+            np.random.multivariate_normal(mean=means[idx], cov=covs[idx], size=sizes[idx])
+        )
+        test_data.append(
+            np.random.multivariate_normal(mean=means[idx], cov=covs[idx], size=sizes[idx])
+        )
+        labels.append(np.array([idx for i in range(sizes[idx])]))
+        test_labels.append(np.array([idx for i in range(sizes[idx])]))
+    X_train = np.vstack(local_data)
+    true_labels_train = np.hstack(labels)
+    X_test = np.vstack(test_data)
+    true_labels_test = np.hstack(test_labels)
+
+    # Compute p(true_label=k)
+    py = np.bincount(true_labels_train) / float(len(true_labels_train))
+
+    noise_matrix = generate_noise_matrix_from_trace(
+        m,
+        trace=avg_trace * m,
+        py=py,
+        valid_noise_matrix=True,
+        seed=seed,
+    )
+
+    # Generate our noisy labels using the noise_matrix for specified number of annotators.
+    s = pd.DataFrame(
+        np.vstack(
+            [generate_noisy_labels(true_labels_train, noise_matrix) for _ in range(num_annotators)]
+        ).transpose()
+    )
+
+    # column of labels without NaNs to test _get_worst_class
+    complete_labels = deepcopy(s)
+
+    # Each annotator only labels approximately 20% of the dataset
+    # (unlabeled points represented with NaN)
+    s = s.apply(lambda x: x.mask(np.random.random(n) < 0.8))
+    s.dropna(axis=1, how="all", inplace=True)
+
+    # Estimate pred_probs
+    latent = count.estimate_py_noise_matrices_and_cv_pred_proba(
+        X=X_train,
+        labels=true_labels_train,
+        cv_n_folds=3,
+    )
+
+    row_NA_check = pd.notna(s).any(axis=1)
+
+    return {
+        "X_train": X_train[row_NA_check],
+        "true_labels_train": true_labels_train[row_NA_check],
+        "X_test": X_test[row_NA_check],
+        "true_labels_test": true_labels_test[row_NA_check],
+        "labels": s[row_NA_check].reset_index(drop=True),
+        "complete_labels": complete_labels,
+        "pred_probs": latent[4][row_NA_check],
+        "noise_matrix": noise_matrix,
+    }
+
+
+def make_data_long(data):
+    data_long = data.stack().reset_index()
+    data_long.columns = ["task", "annotator", "label"]
+
+    return data_long
+
+
+# Global to be used by all test methods. Only compute this once for speed.
+data = make_data()
+
+
+def test_convert_long_to_wide():
+    labels_long = make_data_long(data["labels"])
+    labels_wide = convert_long_to_wide_dataset(labels_long)
+
+    assert isinstance(labels_wide, pd.DataFrame)
+    # TODO: might add a test to see if the wide dataframe actually contains the right info
```

Comment:
how about just checking one index (say 10) of labels_wide contains the proper values from the corresponding index of labels_long?